lang/python/python-paho-mqtt/Makefile

```diff
@@ -0,0 +1,58 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-paho-mqtt
+PKG_VERSION:=1.3.1
+PKG_RELEASE:=1
+PKG_MAINTAINER:=nobody
+PKG_LICENSE:=Eclipse Public License v1.0 / Eclipse Distribution License v1.0
+
+PKG_SOURCE_PROTO:=git
```

Comment:
Could you use codeload instead of a git tag or has?

Reply:
What/where is codeload?  Do I need permissions for it?

Reply:
It's a GitHub service that creates tarballs from tagged versions.  Best thing to do is grep the openwrt packages repo because it's used quite a bit and you can just cut & paste.

Reply:
Let me grab an example...

Reply:
https://github.com/openwrt/packages/blob/561401ad4344d7bfd7f4dda11957cc5d9954897a/net/gitolite/Makefile#L16

Reply:
Wow, you're a fast responder!  I'm afraid that isn't very helpful.  I get the URLs on codeload.github.com that a lot of packages use, but no instructions on how to generate said tarballs.  I don't have permissions to the eclipse repo on github.

Reply:
they're autogenerated if the projects uses tags (if not then it won't help).

Reply:
Oh, docs really helpful, thanks for that.  I've got it bookmarked.  I think I've got it now.  Going to give it a test run.  I don't use github pull requests often, so should I just rebase / squash, test, `git push -f` and submit a new pull request?

```
include $(TOPDIR)/rules.mk

PKG_NAME:=python-paho-mqtt
PKG_VERSION:=1.3.1
PKG_RELEASE:=1
PKG_MAINTAINER:=Daniel Santos <daniel.santos@pobox.com
PKG_LICENSE:=Eclipse Public License v1.0 / Eclipse Distribution License v1.0

PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
PKG_SOURCE_URL:=https://codeload.github.com/eclipse/paho.mqtt.python/tar.gz/v$(PKG_VERSION)?
PKG_HASH:=7cefe65d82acb49be1282294acb12caf7ec86ede9de5b85ea3cc12e8aa6ca7ca

#PKG_SOURCE_PROTO:=git
#PKG_SOURCE_URL:=https://github.com/eclipse/paho.mqtt.python
#PKG_SOURCE_VERSION:=v1.3.1

PKG_SOURCE_SUBDIR:=$(BUILD_VARIANT)-paho-mqtt-$(PKG_VERSION)
PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)-paho-mqtt-$(PKG_VERSION)

include $(INCLUDE_DIR)/package.mk
include ../python-package.mk
include ../python3-package.mk

define Package/python-paho-mqtt/Default
  SUBMENU:=Python
  SECTION:=lang
  CATEGORY:=Languages
  TITLE:=Eclipse Paho MQTT Client Library $(VARIANT)
  MAINTAINER:=Pierre Fersing <pierre.fersing@bleemeo.com>
  URL:=https://www.eclipse.org/paho/clients/python
  DEPENDS:= \
	+PACKAGE_$(VARIANT)-paho-mqtt:$(VARIANT)-light \
	+PACKAGE_$(VARIANT)-paho-mqtt:$(VARIANT)-idna \
	+PACKAGE_$(VARIANT)-paho-mqtt:$(VARIANT)-logging
endef

define Package/python-paho-mqtt
$(call Package/python-paho-mqtt/Default)
  VARIANT:=python
endef

define Package/python3-paho-mqtt
$(call Package/python-paho-mqtt/Default)
  VARIANT:=python3
endef

define Package/python-paho-mqtt/description
  MQTT version 3.1.1 client implementation
endef

define Package/python3-paho-mqtt/description
$(call Package/python-paho-mqtt/description)
.
(Variant for Python3)
endef

$(eval $(call PyPackage,python-paho-mqtt))
$(eval $(call BuildPackage,python-paho-mqtt))
$(eval $(call BuildPackage,python-paho-mqtt-src))

$(eval $(call Py3Package,python3-paho-mqtt))
$(eval $(call BuildPackage,python3-paho-mqtt))
$(eval $(call BuildPackage,python3-paho-mqtt-src))
```

Reply:
No need for  a new PR -- just force-push when ready and ping @commodo (and myself, although I'm just a contributor helping out).